packages/main/src/plugin/contribution-manager.ts

```diff
@@ -78,6 +81,20 @@ export class ContributionManager {
     this.apiSender.send('contribution-register', this.contributions);
   }
 
+  async loadBase64Icon(rootDirectory: string, metadata: any): Promise<string> {
```

Comment:
lint error, no `any` on metadata / something else on it?

Reply:
`metadata` is a raw representation of package.json, I think, we can add
`// eslint-disable-next-line @typescript-eslint/no-explicit-any`
as it has been done for `loadMetadata` method, wdyt?

Reply:
added a comment to the line